<launch>

<arg name="rate" default="1"/>

<node name="pointcloud_loader" pkg="pcl_ros" type="pcd_to_pointcloud" args="/media/justin/data/real_testing/rtabmap_records/cloud_1_4.pcd">
  <param name="frame_id" value="/odom"/>
</node>


<include file="$(find nav_scripts)/launch/playbag.launch" >
  <arg name="bag_files" value=" /media/justin/data/real_testing/rtabmap_records/processed_1_4.bag"/>
  <arg name="rate" value="$(arg rate)"/>
</include>

<node name="map" pkg="map_server" type="map_server" args="$(find nav_configs)/maps/fourth_floor_gazebo.yaml"/>

</launch>
